### PR TITLE
Cassandra 17570 4.1

### PR DESCRIPTION
**Pull request by bereng**

### CODE REVIEW

**Comment by bereng** [comment]
CI [j8](https://app.circleci.com/pipelines/github/bereng/cassandra/653/workflows/f36f4fa2-2973-4ec6-9142-89a047f9bd40)
CI [j11](https://app.circleci.com/pipelines/github/bereng/cassandra/653/workflows/a000224b-f418-4bbc-ba69-154f0b95723f)

File: README.asc [hunk 1]
```diff
@@ -39,7 +39,7 @@ be sitting in front of a prompt:
 
 ----
 Connected to Test Cluster at localhost:9160.
-[cqlsh 6.0.0 | Cassandra 4.0.2 | CQL spec 3.4.5 | Native protocol v5]
```
Comment (bereng) [inline README.asc:42]: Mmmm this should be `Cassandra 4.0.2` -> `Cassandra 4.1`. I can do that on commit though.

**Review by blerer — APPROVED** [review]

**Comment by bereng** [comment]
CI [j11](https://app.circleci.com/pipelines/github/bereng/cassandra/686/workflows/36c1c1cd-a172-4c05-b006-fdc12ea945fa)
CI [j8](https://app.circleci.com/pipelines/github/bereng/cassandra/686/workflows/d4aaaa64-0d1a-45ac-a654-31009206259b)

**Comment by bereng** [comment]
Ran upgrade tests for extra safety [here](https://app.circleci.com/pipelines/github/bereng/cassandra/689/workflows/b5f494da-372c-4432-9be2-4e1dea144d46)

**Pull request closed without merging**